doc/source/myrocks/variables.rst

```diff
@@ -1984,6 +1992,31 @@ to the info log.
 Default value is ``600``.
 Allowed range is up to ``2147483647``.
 
+.. variable:: rocksdb_stats_level
+
+  :version 5.7.30-33: Implemented
+  :cli: ``--rocksdb-stats-level``
+  :dyn: Yes
+  :scope: Global
+  :vartype: Numeric
+  :default: ``0``
+
+Controls the RocksDB statistics level. The default value is "0" (kExceptHistogramOrTimers),
+ which is the fastest level. The maximum value is "4".
+
+.. variable:: rocksdb_stats_recalc_rate
+
+  :version 5.7.23-23: Implemented
```

Comment:
8.0.x
This was not ported to 8.0 before?

Reply:
Removed 5.7 reference